### PR TITLE
Cinebot QuickStart Guide for BUILD2024

### PR DESCRIPTION
**Pull request by sfc-gh-cyorek**
Pushing quickstart code for BUILD 2024 HOL Session

### CODE REVIEW

File: site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
+author: Charles Yorek
+id: build-a-cinebot-with-snowflake-native-apps-and-cortex
+summary: This is a sample Snowflake Guide
+categories: Getting-Started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Native Apps, Cortex, Data Applications
+
+# Build a CineBot with Snowflake Native Apps and Cortex
```
Comment (iamontheinet) [inline site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md]: @sfc-gh-cyorek Pl change the title to "Build a CineBot with Snowflake Native Apps and Snowfake Cortex"

File: site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+<!-- ------------------------ -->
+## Overview
+Duration: 1
+
+The Snowflake Native App Framework is a powerful way for application providers to build, deploy and market applications via the Snowflake Marketplace.  In this example you will learn how to incorporate Snowflake's Cortex Suite into a Native App that will create a 'CineBot' that can recommend Movies. 
+
+### Prerequisites
```
Comment (iamontheinet) [inline site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md:17]: @sfc-gh-cyorek Add link to Snowflake signup/signin page.

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md (hunk 1 above)
Comment (sfc-gh-cyorek) [reply site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md]: Updated

File: site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md (hunk 2 above)
Comment (sfc-gh-cyorek) [reply site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md:17]: Added

**Pull request merged**